src/StoreClient.h

```diff
@@ -114,6 +119,11 @@ class store_client
     int type;
     bool object_ok;
 
+    StoreIOBuffer copyInto;
+    /// the number of bytes effectively copied from Store into the I/O buffer
+    size_t copiedSize;
```

Comment:
Please remind me what we mean by "effectively" here.

Reply:
IIRC there can be a situation when some bytes were copied into the buffer (copyInto), but an error occurred. In this case, copiedSize is still zero, i.e., it describes what bytes are 'effectively' available from the I/O buffer.